environments/icds-cas/inventory/citusdb.csv

```diff
@@ -1,61 +1,61 @@
-citusdb_master,postgresql_max_wal_size,8GB,S,,
-citusdb_master,postgresql_max_worker_processes,32,I,,
-citusdb_master,postgresql_min_wal_size,4GB,S,,
-citusdb_master,postgresql_random_page_cost,1.1,F,,
-citusdb_master,postgresql_shared_buffers,20GB,S,,
-citusdb_master,postgresql_wal_buffers,16MB,S,,
-citusdb_master,postgresql_work_mem,1872kB,S,,
-citusdb_worker,pgbouncer_default_pool,450,I,,
-citusdb_worker,postgresql_checkpoint_completion_target,0.9,F,,
-citusdb_worker,postgresql_default_statistics_target,500,I,,
-citusdb_worker,postgresql_effective_cache_size,60GB,S,,
-citusdb_worker,postgresql_effective_io_concurrency,200,I,,
-citusdb_worker,postgresql_maintenance_work_mem,2GB,S,,
-citusdb_worker,postgresql_max_connections,500,I,,
-citusdb_worker,postgresql_max_parallel_workers_per_gather,16,I,,
-citusdb_worker,postgresql_max_wal_size,8GB,S,,
-citusdb_worker,postgresql_max_worker_processes,32,I,,
-citusdb_worker,postgresql_min_wal_size,4GB,S,,
-citusdb_worker,postgresql_random_page_cost,1.1,F,,
-citusdb_worker,postgresql_shared_buffers,20GB,S,,
-citusdb_worker,postgresql_wal_buffers,16MB,S,,
-citusdb_worker,postgresql_work_mem,1872kB,S,,
+hostname,host_address,group 1,group 2,group 3,var: hostname,H.var: hot_standby_server,H.var: hot_standby_master,L.var: postgresql_replication_slots,var: replication_slot
+citusmaster0,100.71.184.115,citusdb,citusdb_master,,MUMGCCWCDPRDCC00,citusmaster0standby,,"[""standby0""]",
+citusworker0,100.71.184.84,citusdb,citusdb_worker,,MUMGCCWCDPRDCW00,citusworker0standby,,"[""standby0""]",
+citusworker1,100.71.184.81,citusdb,citusdb_worker,,MUMGCCWCDPRDCW01,citusworker1standby,,"[""standby0""]",
+citusworker2,100.71.184.122,citusdb,citusdb_worker,,MUMGCCWCDPRDCW02,citusworker2standby,,"[""standby0""]",
+citusworker3,100.71.184.118,citusdb,citusdb_worker,,MUMGCCWCDPRDCW03,citusworker3standby,,"[""standby0""]",
+citusworker4,100.71.184.126,citusdb,citusdb_worker,,MUMGCCWCDPRDCW04,citusworker4standby,,"[""standby0""]",
+citusworker5,100.71.184.125,citusdb,citusdb_worker,,MUMGCCWCDPRDCW05,citusworker5standby,,"[""standby0""]",
+citusworker6,100.71.184.68,citusdb,citusdb_worker,,MUMGCCWCDPRDCW06,citusworker6standby,,"[""standby0""]",
+citusworker7,100.71.184.127,citusdb,citusdb_worker,,MUMGCCWCDPRDCW07,citusworker7standby,,"[""standby0""]",
+citusworker8,100.71.184.143,citusdb,citusdb_worker,,MUMGCCWCDPRDCW08,citusworker8standby,,"[""standby0""]",
+citusworker9,100.71.184.132,citusdb,citusdb_worker,,MUMGCCWCDPRDCW09,citusworker9standby,,"[""standby0""]",
+citusworker10,100.71.184.116,citusdb,citusdb_worker,,MUMGCCWCDPRDCW10,citusworker10standby,,"[""standby0""]",
+citusworker11,100.71.184.123,citusdb,citusdb_worker,,MUMGCCWCDPRDCW11,citusworker11standby,,"[""standby0""]",
+citusmaster0standby,100.71.184.92,citusdb,citusdb_master,pg_standby,MUMGCCWCDPRDCC01,,citusmaster0,,standby0
+citusworker1standby,100.71.184.128,citusdb,citusdb_worker,pg_standby,MUMGCCWCDPRDCW12,,citusworker0,,standby0
```

Comment:
We should rename these to have consistent numerals, if it's not too late. It will be confusing in future.

Reply:
Sorry for the delayed comment, though. I think it will just need deploy with tags `hostname` and `datadog`